mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -248,7 +248,14 @@ def _init_settings(self):
 
         # migrate settings if needed
         if not exists(self._settings_path) and exists(self._old_settings_path):
+            LOG.warning("Found skill settings at pre-xdg location, migrating!")
             shutil.copy(self._old_settings_path, self._settings_path)
+            LOG.info(f"{self._old_settings_path} moved to {self._settings_path}")
+        if not exists(self._settings_path) and exists(self._old_xdg_settings_path):
```

Comment:
we should probably remove this in final 0.0.2 or 0.1.0 release since this bug is not present in mycroft and was only around in alphas